renku/cli/service.py

```diff
@@ -193,6 +197,13 @@ def service(ctx, env):
 
         ctx.exit(1)
 
+    except redis.exceptions.ConnectionError:
+        # NOTE: Cannot connect to the service dependencies, ie. Redis.
+
+        click.echo(ERROR + "Cannot connect to Redis")
```

Comment:
What do the `ConnectionError`s look like? If they contain any useful information that could help a user to solve an issue, we might want to include the original error in this message.

Reply:
It just says that redis could not be reached, nothing too useful.